packages/angular/cli/commands/e2e.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/e2e.json",
   "description": "Builds and serves an Angular app, and runs end-to-end tests using Protractor.",
-  "$longDescription": "Must be executed from within a workspace directory.\n When a project name is not supplied, the configured default e2e project of the workspace is used.",
+  "$longDescription": "./e2e-long.md",
```

Comment:
So was this the error that caused the output to be invalid?

Reply:
Yes, that was the one. That should be a file path. We output a warning instead of failing before. 

The warning would actually happen on every `ng` command, not just the help ones.